doc/introduction/measurements.rst

```diff
@@ -291,7 +291,8 @@ PennyLane device capabilities.
 >>> transformed_qnode(*pars)
 tensor([0.90165331, 0.09834669], requires_grad=True)
 
-The decorator syntax applies equally well:
+The decorator syntax applies equally well, and finally, ``defer_measurements`` may
+be applied to a QNode directly, too:
```

Comment:
The description sounded like it was not possible to apply the transform to QNodes but only to quantum functions. In my opinion we should display both options.

Reply:
Can't make a suggestion, but here's what I would say instead:

"The decorator ``@qml.defer_measurements`` applies equally well to both QNodes and quantum functions:"

Reply:
I like the phrasing! I would want to make sure in addition that `defer_measurements` can be applied to a QNode whether it is used as a decorator or not (might not be clear to users).
Maybe
"qml.defer_measurements applies equally well as decorator and functional call, and to both QNodes and quantum functions:"
?

Reply:
We could alternatively just focus on `defer_measurements` being applied to a QNode, I don't think we need to worry about both ways around.